packages/discovery-react-components/src/components/SearchResults/components/Result/__tests__/Result.test.tsx

```diff
@@ -3,743 +3,547 @@ import React from 'react';
-      (context.searchResponseStore!.data as DiscoveryV2.QueryResponse).results = [
+  describe('when there is a QueryResult available in searchResponseStore', () => {
+    beforeEach(() => {
+      queryResults = [
         {
           document_id: 'some document_id',
-          result_metadata: {
-            collection_id: '1'
-          },
-          document_passages: [
-            {
-              passage_text: 'this is the passage text'
-            }
-          ]
+          text: 'body text'
         }
       ];
-      const api = {
-        setSelectedResult: mockSelectResult
-      };
-      const { getByText } = render(wrapWithContext(<SearchResults />, api, context));
-      fireEvent.click(getByText('View passage in document'));
-      expect(mockSelectResult.mock.calls.length).toBe(1);
-      expect(mockSelectResult.mock.calls[0][0].document).toBe(
-        context.searchResponseStore!.data!.results![0]
-      );
-      expect(mockSelectResult.mock.calls[0][0].element).toBe(
-        context.searchResponseStore!.data!.results![0].document_passages![0]
-      );
-      expect(mockSelectResult.mock.calls[0][0].elementType).toBe('passage');
     });
 
-    test('will call onSelectResult with result and table element and element type as parameters', () => {
-      const mockSelectResult = jest.fn();
-      (context.searchResponseStore!.data as DiscoveryV2.QueryResponse).results = [
-        {
-          document_id: 'some document_id',
-          result_metadata: {
-            collection_id: '1'
-          },
-          document_passages: [
-            {
-              passage_text: 'this is the passage text'
-            }
-          ]
-        }
-      ];
-      (context.searchResponseStore!.data as DiscoveryV2.QueryResponse).table_results = [
-        {
-          table_id: '558ada041262d5b0aa02a05429d798c7',
-          source_document_id: 'some document_id',
-          collection_id: '8713a92b-28aa-b291-0000-016ddc68aa2a',
-          table_html:
-            '<table style="width:100%"><tr><th>Firstname</th><th>Lastname</th><th>Age</th></tr><tr><td>Jane</td><td>Smith</td><td>50</td></tr><tr><td>Eve</td><td>Jackson</td><td>94</td></tr></table>'
-        }
-      ];
-      const api = {
-        setSelectedResult: mockSelectResult
-      };
-      const { getByText } = render(wrapWithContext(<SearchResults />, api, context));
-      fireEvent.click(getByText('View table in document'));
-      expect(mockSelectResult.mock.calls[0][0].document).toBe(
-        context.searchResponseStore!.data!.results![0]
-      );
-      expect(mockSelectResult.mock.calls[0][0].element).toBe(
-        context.searchResponseStore!.data!.table_results![0]
-      );
-      expect(mockSelectResult.mock.calls[0][0].elementType).toBe('table');
+    describe('and usePassages is set to true', () => {
+      beforeEach(() => {
+        componentProps.usePassages = true;
+      });
+
+      test('will render the bodyField', () => {
```

Comment:
Should this clarify that it will render the bodyField in this case if the first passage doesn't exist?